pkg/cmd/service/service.go

```diff
@@ -107,8 +107,24 @@ type PlaygroundConfig struct {
 	Port    int
 }
 
-// OpenFGAConfig defines server configurations specific to the OpenFGA server itself.
-type OpenFGAConfig struct {
+// ProfilerConfig defines server configurations specific to pprof profiling.
+type ProfilerConfig struct {
+	Enabled bool
+	Addr    string
+}
+
+type Config struct {
+	// If you change any of these settings, please update the documentation at https://github.com/openfga/openfga.dev/blob/main/docs/content/intro/setup-openfga.mdx
+
+	// ListObjectsDeadlineInSeconds defines the maximum amount of time to accumulate ListObjects results
+	// before the server will respond. This is to protect the server from misuse of the
+	// ListObjects endpoints.
+	ListObjectsDeadlineInSeconds int
```

Comment:
Why not duration? It's a duration type. If, at the command line, I want to specify `--listObjects-deadline=3ms` then I should be able to do that. Deadlines don't have to be restricted to seconds if I want a strict deadline.

Reply:
changed :)